.github/workflows/merge_request_app_id.yml

```diff
@@ -0,0 +1,32 @@
+name: Validate CSR Common Name
+on:
+  pull_request:
+    paths:
+      - '**/*.csr'
+
+jobs:
+  validate-csr-cn:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
can you replace it with the hash ?
otherwise that is a security issue as it happened that tag got compromised

Reply:
@mgallien thank you for your tip. I added the hash and update to action version to v5